### ISSUE TITLE
Does not work.

### ISSUE THREAD

**Issue by joseywoermann** [joseywoermann/kgh-notifier#3]
I don't know why.

**Comment by byZeroOfficial** [comment]
Do you found a solution?

**Comment by joseywoermann** [comment]
@byZeroOfficial No, but I have been using [this](https://play.google.com/store/apps/details?id=me.webalert) app for a while now and it works great.
